app/controllers/admin/accounts_controller.rb

```diff
@@ -34,6 +34,15 @@ def silence
     redirect_to admin_accounts_path
   end
 
+  def timeout
+    @account.update(silenced: true)
+    UnsilenceWorker.perform_in(
+      (params[:time].to_i * params[:modifier].to_i).hours,
+      @account.id
+    )
+    redirect_to admin_accounts_path
```

Comment:
I feel like this is a dangerous way to mange unsilencing. If a job fails (and there are many reasons it could fail) a user could potentially never be unsilenced